modules/stochastic_tools/src/outputs/MappingOutput.C

```diff
@@ -0,0 +1,52 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Moose includes
+#include "MappingOutput.h"
+#include "SurrogateTrainer.h"
+#include "FEProblem.h"
+#include "RestartableDataIO.h"
+#include "RestartableData.h"
+
+registerMooseObject("StochasticToolsApp", MappingOutput);
+
+InputParameters
+MappingOutput::validParams()
+{
+  InputParameters params = FileOutput::validParams();
+  params.addClassDescription("Output for mapping model data.");
+  params.addRequiredParam<std::vector<UserObjectName>>("mappings",
```

Comment:
this is really not good. a mapping is not a UO

Reply:
changed to string!